amazonia-1/amazonia-1-tiler-example.ipynb

```diff
@@ -0,0 +1,721 @@
+{
```

Comment:
<p>See the comments on <a href="https://github.com/developmentseed/asdi-examples/pull/12" target="_blank">https://github.com/developmentseed/asdi-examples/pull/12</a></p><p>We changed a few things in the text, like the heading of this section </p><h3>Overview Map for Context</h3><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/asdi-examples/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>